src/components/BaseComponents/BaseWeatherIcon.tsx

```diff
@@ -1,10 +1,14 @@
 "use client";
 type BaseWeatherIconProps = {
-    iconImg: string;
-    isDay: boolean;
+    weatherDescriptionText: string;
+    assetId: string;
 };
 
-export default function BaseWeatherIcon({ iconImg, isDay }: Readonly<BaseWeatherIconProps>) {
-    const renderImg = `weather_conditions/${isDay ? "day" : "night"}/${iconImg}`;
-    return <img src={`${renderImg}.svg`} alt="Weather icon" className="!w-full h-full" />;
+export default function BaseWeatherIcon({ weatherDescriptionText, assetId }: Readonly<BaseWeatherIconProps>) {
+    const imagePath = `${process.env.NEXT_PUBLIC_APP_BASE_URL}assets/${assetId}`;
```

Comment:
I noticed we set this in a couple of places. I'm thinking if it's worth setting it as a constant regex, or move this in the DataService as static and import it from there

Reply:
Good idea regarding this. I don't believe it belongs to DataService since this is not an actual API call, like the rest of them. Maybe add it to `src/app/appConfig.ts` and export the path from there? Or it could be a function stored in common folder which takes the assetId and returns the full path maybe?

Reply:
Helper function introduced.